apps/studio/src/features/editing-experience/components/CreateCollectionPageModal/CreateCollectionPageWizardContext.tsx

```diff
@@ -93,7 +95,8 @@ const useCreateCollectionPageWizardContext = ({
       },
       {
         onSuccess: ({ pageId }) => {
-          void router.push(`/sites/${siteId}/pages/${pageId}`)
+          const nextType = type === "page" ? "pages" : "links"
```

Comment:
nit: I think this routes might need to be defined as a constant somewhere, especially since we have `folders`, `collections`, `pages` and now `links`.

Reply:
actually this is already defined as `ResourceType`

Reply:
Ah found it, this should be using `getResourceSubpath` so we can be consistent.